regression-test/suites/inverted_index_up_down_p0/load.groovy

```diff
@@ -0,0 +1,93 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_upgrade_downgrade_prepare_inverted_index","p0,inverted_index,restart_fe") {
+    for (version in ["V1", "V2"]) {
+        sql "drop table if exists t_up_down_inverted_index${version}_duplicate"
+        sql String.format('''
+        create table t_up_down_inverted_index%s_duplicate(
+        a int,
+        b int,
+        c int,
+        index idx_a(a)
```

Comment:
inverted_index should contain fulltext with tokenized, fulltext without tokenized and numerical value.